Frontend/src/test/pages/dashboard/components/Customer/EditAddressDialog.test.jsx

```
@@ -0,0 +1,116 @@
+    DialogTitle: ({ children }) => <h1 data-testid="title">{children}</h1>,
+  })
+)
+
+// Stub AddressForm to simulate form submission
+vi.mock(
+  '../../../../../components/ui/input',
+  () => ({ Input: props => <input {...props} /> })
+)
+vi.mock(
+  '../../../../../components/ui/label',
+  () => ({ Label: props => <label {...props} /> })
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding an accessible text content to the label element.
Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)